src/Interfaces/DynamoTestStation.ts

```diff
@@ -15,4 +17,37 @@ interface DynamoTestStation {
   testStationType: string;
 }
 
-export { DynamoTestStation };
+const TestStationType = new Map<number, string>([
+  [147160000, 'ATF'],
+  [147160001, 'IVA & ATF'],
+  [147160002, 'IVA'],
+]);
+
+const TestStationStatus = new Map<number, string>([
```

Comment:
I wonder if we want a strongly typed version of this?

const Pending = 147160000
const Active = 147160001
...

const TestStationStatuses = Pending | Active | etc